### PR TITLE
refactor: moved analyses link

### PR DESCRIPTION
**Pull request by Ryoken**
#### What's this PR do?
- moves analyses link to sidebar
- adjusts navigation of analisi pages

#### How should this be manually tested?
Click through nav with analyses.

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/2863

#### Screenshots (if appropriate)
Link on main nav:
Screen Shot 2021-09-14 at 4 40 20 PM

Clicking on an analyses brings you to it's page with a link back to the analyses list:
Screen Shot 2021-09-14 at 4 40 29 PM

Clicking on a run id brings you to the detail page with a link back to the analysis:
Screen Shot 2021-09-14 at 4 40 38 PM

### CODE REVIEW

File: seed/templates/seed/_sidebar.html [hunk 1]
```diff
@@ -29,6 +29,10 @@
             <div class="item_name" translate>Organizations</div>
             <div ng-show="logged_in" class="badge badge_menu">{$ organizations_count | number:0 $}</div>
         </a>
+        <a id="sidebar-accounts" ng-class="{item: true, active: is_active('/analyses/' + menu.user.organization.id), 'disabled-item': !logged_in}" title="{$:: 'Analyses' | translate $}" href="{% url "seed:home" %}#/analyses/{$:: menu.user.organization.id $}">
```
Comment (macintoshpie) [inline seed/templates/seed/_sidebar.html:32]: I'm not sure how to fix this, but it looks like the org id isn't being updated when I change update it so the analyses link always goes to the analyses for the same org until I refresh the page.

Screen Shot 2021-09-15 at 12 14 31 PM


I think there's some kind of refresh event happening when the user updates the selected org, not sure how to integrate it with the sidebar though.

I think we also need to handle the case where the user is on the analyses page and they change the active org (it should refresh the analyses page for the new org). Again this is handled by some kind of refresh on other pages (e.g. try going to a property detail page then changing the org) but I don't know much else about it.

**Review by macintoshpie — COMMENTED** [review]
Looks nice! One comment on refreshing the nav/page when the org changes. Also I think we need to update this link:
https://github.com/SEED-platform/seed/blob/2863-refactor/move-analyses-link/seed/static/seed/js/controllers/inventory_list_controller.js#L1059

File: seed/templates/seed/_sidebar.html (hunk 1 above)
Comment (Ryoken) [reply seed/templates/seed/_sidebar.html:32]: Good catch.  I'll take a look!

Comment (Ryoken) [reply seed/templates/seed/_sidebar.html]: I've refactored the analyses pages to update when the org changes.  The org ID is no longer a URL param and the link behaves like the other nav links, getting org id from the user_service instead.

**Review by macintoshpie — APPROVED** [review]
💯 🔥

**Pull request merged**